orchagent/dash/dashvnetorch.cpp

```diff
@@ -283,10 +285,67 @@ void DashVnetOrch::addOutboundCaToPa(const string& key, VnetMapBulkContext& ctxt
     auto& object_statuses = ctxt.outbound_ca_to_pa_object_statuses;
     sai_attribute_t outbound_ca_to_pa_attr;
     vector<sai_attribute_t> outbound_ca_to_pa_attrs;
+    
+    DashOrch* dash_orch = gDirectory.get<DashOrch*>();
+    dash::route_type::RouteType route_type_actions;
+    if (!dash_orch->getRouteTypeActions(ctxt.metadata.routing_type(), route_type_actions))
+    {
+        SWSS_LOG_INFO("Failed to get route type actions for %s", key.c_str());
+    }
 
-    outbound_ca_to_pa_attr.id = SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_UNDERLAY_DIP;
-    to_sai(ctxt.metadata.underlay_ip(), outbound_ca_to_pa_attr.value.ipaddr);
-    outbound_ca_to_pa_attrs.push_back(outbound_ca_to_pa_attr);
+    for (auto action: route_type_actions.items())
+    {
+        if (action.action_type() == dash::route_type::ACTION_TYPE_STATICENCAP)
```

Comment:
@theasianpianist, UNDERLAY_DIP is not populated for static encap action. This will break the non PL cases.